content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,507 @@
+entire appraisal loads and can be clicked through without having to enter any answers. To exit,
+close the tab.
+
+To view the results from everyone who completed an appraisal, click the :guilabel:`See Results`
+button. This presents all the answers for the survey in a new tab. Each question provides
+information on how many people responded to a question, and how many people skipped it. All answers
+for each question are visible. To exit, close the tab.
+
+.. image:: appraisals/survey-list.png
+   :align: center
+   :alt: A list view of all available surveys in the Appraisals application.
+
+In addition to viewing the responses from past appraisals and surveys, new surveys can also be
+created from this form. Click the :guilabel:`New` button at the top to create a new survey. For more
+information on how to create a survey, refer to the :doc:`../marketing/surveys/create` document.
+
+.. note::
+   In previous versions of Odoo, this section was referred to as :guilabel:`Surveys`.
+
+Tags
+----
+
+An additional feature of the *Appraisals* application is the ability to set :ref:`goals
+<appraisals/goals>` for employees. On each goal form, there is a field to add tags. The available
+tags are pulled from a list that is configured from the settings.
+
+To view a list of all existing goal tags, and to create new tags, navigate to
+:menuselection:`Appraisals application --> Configuration --> Tags`.
+
+All currently configured tags are presented in a list view. The *Appraisals* application does not
+have any pre-configured tags, so all tags need to be added either from this list, or directly from
+the goal form.
+
+To create a new tag, click the :guilabel:`New` button, and a blank line appears. Enter the name of
+the tag on the line. Press the enter key to save the tag and create a new blank line. Repeat this
+for all tags that need to be added. The list of tags is automatically arranged in alphabetical
+order.
+
+Appraisals
+==========
+
+Navigate to the main appraisals dashboard by opening the *Appraisals* application. The appraisals
+dashboard is the default view. To view the dashboard at any point in the application, navigate to
+:menuselection:`Appraisals application --> Appraisals`.
+
+All appraisals are displayed on the dashboard in a kanban view, with a list of groupings on the left
+side of the dashboard, including :guilabel:`Company`, :guilabel:`Department`, and
+:guilabel:`Status`. Click any grouping option to view appraisals for **only** the chosen selection.
+
+.. note::
+   Only groupings with multiple selections appear in the list. For example, if a database only has
+   one company, the :guilabel:`Companies` grouping does not appear, since there is no other company
+   to select.
+
+Each card displays the following information about the appraisal:
+
+- :guilabel:`Name`: the employee's name.
+- :guilabel:`Department`: the department the employee is part of.
+- :guilabel:`Company`: the company the employee works for.
```

Comment:
This only appears in a multi-company scenario. if there's only one, then there's nothing to select since they are part of the single company. I'll add a note!